src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public void getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColorNumber();
+        System.out.println(ball.toString());
```

Comment:
It seems like you are not returning any value from the `getRandomBall` method. According to best practices, if a method is named to suggest that it retrieves something (like 'getRandomBall'), it should return the value it gets. Consider changing the method to return a Ball object instead of printing it within the method.